src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue8291.cs

```diff
@@ -1,4 +1,5 @@
-﻿using NUnit.Framework;
+﻿#if TEST_FAILS_ON_CATALYST // TouchAndHold not supported on Mac, Also using LongPress is not applicable on this case, while LongPress does not open the context menu for Entry/Editor control.
```

Comment:
Could you try to use `App.RightClick`?

Reply:
@jsuarezruiz , When manually testing the sample, right-clicking opens the context menus. However, during automation, while the App.RightClick function executes, the context menu does not appear. Nonetheless, the test passes because the copy, paste, and edit buttons are accessible in the macOS app menu bar. I have attached video for reference, can you please take a look and share your suggestions.
Mannual Test:

![image](https://github.com/user-attachments/assets/0204be8e-4f26-49ae-ab53-9527c9ac09de)

Automation Test:


https://github.com/user-attachments/assets/6362af2f-f6c3-42d9-beaa-041a14768257